src/main/java/io/fabric8/jenkins/openshiftsync/ConfigMapWatcher.java

```diff
@@ -190,6 +196,46 @@ private boolean containsSlave(ConfigMap configMap) {
                 podTemplate = xStream2.fromXML(entry.getValue());
 
                 if( podTemplate instanceof PodTemplate ) {
+                    PodTemplate pt = (PodTemplate)podTemplate;
+                    
+                    // if the image field is and imagestream related ref, without the internal docker registry, prepend it
+                    String image = pt.getImage();
+                    try {
+                        if (!OpenShiftUtils.hasRegistry(image)) {
```

Comment:
not sure you followed my last suggestion.

i'm suggesting we allow the user to supply something like:
`<image>imagestreamtag:myproject/myimage:mytag</image>`

so they are telling us, in the image field, whether the value is an image value (no prefix) or an imagestreamtag value (prefixed with imagestreamtag)